src/views/AddItem.jsx

```diff
@@ -1,3 +1,77 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [newItem, setNewItem] = useState('');
+	const [nextPurchaseTime, setPurchaseTime] = useState(7);
+	const [statusMessage, setStatusMessage] = useState('');
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		let listId = 'my test list';
+
+		const item = await addItem(listId, {
+			itemName: newItem,
+			daysUntilNextPurchase: nextPurchaseTime,
+		});
+
+		if (item.id) {
+			setStatusMessage('Item Successfully Added!');
```

Comment:
once you add an item, the success message is always present in the UI. Maybe you can add a timeout to make the message disappear after some seconds.